tfhe/src/boolean/public_key/compressed.rs

```diff
@@ -95,9 +95,9 @@ mod tests {
     };
     use crate::boolean::random_boolean;
     #[cfg(not(feature = "__coverage"))]
-    const NB_TEST: usize = 32;
```

Comment:
Why the s ?

Reply:
For consistency, it's with an `s` in other files